packages/vscode-extension/src/utils/localizeUtils.ts

```diff
@@ -71,15 +71,15 @@ function getLocalizedString(key: string, defValue?: string): string {
 export function loadLocalizedStrings(): void {
   loadedLocale = parseLocale();
 
-  const nlsFile = path.join(EXTENSION_ROOT_DIR, "..", `package.nls.${loadedLocale}.json`);
+  const nlsFile = path.join(ext.context.extensionPath, `package.nls.${loadedLocale}.json`);
```

Comment:
Use extension context instead of __dirname since the folder structure changed after webpack. It works when local debug because we use tsc for local debug and webpack is just for the webviews.